.phan/config.php

```diff
@@ -357,6 +357,7 @@
     // your application should be included in this list.
     'directory_list' => [
         'api',
+        'Context',
```

Comment:
So... the context is part of neither the API nor sdk?

Reply:
That's correct.  The [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/context/context.md) has correlation living as its own thing.  Context is implemented at a language level in many other languages, so they were attempting to decouple the Context implementation from CorrelationContext so that authors could either write their own Correlation implementation or adapt their languages' Correlation packages.  The CorrelationContext is part of the API.